### PR TITLE
add ppid for S1AP

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/s1ap/sctp_conn.c [hunk 1]
```diff
@@ -101,8 +101,10 @@ int recv_sctp_msg(int connSock, unsigned char *buffer, size_t len)
 
 int send_sctp_msg(int connSock, unsigned char *buffer, size_t len)
 {
+	uint32_t ppid = 0x12000000;
```
Comment (krsna1729) [inline src/s1ap/sctp_conn.c]: Explain this magic number

File: src/s1ap/sctp_conn.c [hunk 2]
```diff
@@ -101,8 +101,10 @@ int recv_sctp_msg(int connSock, unsigned char *buffer, size_t len)
...
 	return sctp_sendmsg(connSock, (void *)buffer, len,
-			NULL, 0, 0, 0, 0, 0, 0);
+			//NULL, 0, 0, 0, 0, 0, 0);
```
Comment (krsna1729) [inline src/s1ap/sctp_conn.c]: Remove if not required

**Review by krsna1729 — COMMENTED** [review]
Needs commit message

File: src/s1ap/sctp_conn.c (hunk 1 above)
Comment (badhrinathpa) [reply src/s1ap/sctp_conn.c]: Better to add a macro for this.

**Review by badhrinathpa — COMMENTED** [review]
+1

File: src/s1ap/sctp_conn.c (hunk 1 above)
Comment (hyunsun) [reply src/s1ap/sctp_conn.c]: #define S1AP_SCTP_PPID (18)
htonl(S1AP_SCTP_PPID)

seems work.

**Comment by hyunsun** [comment]
+1, need this patch.

**Pull request merged**